src/user/userController.js

```diff
@@ -18,7 +18,58 @@ exports.addUser = async (req, res) => {
           res.status(200).send({ message: `Successfully deleted ${searchUser.username}`});
       } catch (error) {
         console.log(error);
-        res.status(500).send({ message: "Unsuccessful, please try again later" });
+        res.status(500).send({ message: "Unsuccessful, please try again later (Delete user)" });
       }
   }
 
+  exports.listUsers = async (req, res) => {
+    try {
+      const user = await User.find({});
```

Comment:
can we change it to allUsers or users because it would return a lists of users not just one user